hbase-server/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupInfoManagerImpl.java

```diff
@@ -1192,6 +1192,10 @@ public BalanceResponse balanceRSGroup(String groupName, BalanceRequest request)
         return responseBuilder.build();
       }
 
+      if (request.isReloadConfigs()) {
```

Comment:
Thanks for the review @Apache9. I could do this, but I didn't do it because currently we need to implement the clone and reload logic twice: here and in HMaster.java. Would you prefer that I implement it twice?

Reply:
Introduce a util method called reloadBalancerConfig and add some comments to say it? Just want to make the balancer code itself clean.